### ISSUE TITLE
USB device on Windows

### ISSUE THREAD

**Issue by taHC81** [PurpleAlien/must-power_grafana#1]
Hello, can I use this script somehow on Windows as well? What should be the device name instead of **/dev/ttyUSB0**?
`
minimalmodbus.Instrument('/dev/ttyUSB0', 10)`

**Comment by PurpleAlien** [comment]
Yes, should work on Windows. You should pass the appropriate COM port there, for example COM4.

**Comment by taHC81** [comment]
Thanks, I've tried, but no device found. I've got MUST PV1800 inverter, connected via RS485 -> TTL -> USB adapter, changed the baud rate to 19200 and tried several slave addresses...

![image](https://user-images.githubusercontent.com/36913630/208254628-a7ca306a-3b51-43e2-a60c-b34c6efa532f.png)

Any suggestion?

**Comment by PurpleAlien** [comment]
So according to the screenshot, the address is 4 and baudrate 19200, which I assume you tried. 
Can you try setting the timeout to 0.5 or 0.6?

**Comment by taHC81** [comment]
Yes, I tried it that way, buy no success, it's a different protocol. I found this one working out of the box - https://github.com/andremiller/must-inverter-python-monitor

**Comment by PurpleAlien** [comment]
Ah, right. Ok. I'll see if it makes sense to add multiple devices to mine. I was thinking you had issues with your com port/serial itself, but yeah, I did not realize it used a different protocol.
